.github/CODEOWNERS

```diff
@@ -5,9 +5,9 @@
 
 # Secondary repo maintainers, substitutes of the primary
 # maintainers when they become MIA
-*       @cwgoes @sunnya97
+*       @cwgoes @marbar3778 @ebuchman @robert-zaremba @amaurym @fedekunze @odeke-em
```

Comment:
In my opinion, @fdymylja and @jgimeno should be part of this list. I believe they both earned our trust and confidence.

Reply:
👍

Reply:
Please ignore my comment Billy and press ahead